src/screens/Customers/CustomerProfile.tsx

```diff
@@ -36,17 +32,20 @@ const styles = (theme: Theme) =>
     buttonPrimary: {
       borderRadius: '12px',
       color: 'white',
+      marginRight: '5px',
       backgroundColor: theme.palette.primary.main,
     },
-    buttonSecondary: {
-      borderRadius: '12px',
-      color: 'white',
-      backgroundColor: theme.palette.primary.main,
+    meterInfoGrid: {
+      display: 'flex',
+    },
+    meterInfoCol: {
+      width: '50%',
+      padding: '5px',
```

Comment:
This padding is also causing this grid to be slightly inset. I couldn't come up with an immediate solution but I've consulted  @mylesdomingo
![image](https://user-images.githubusercontent.com/21160510/114511070-48bf0f00-9bec-11eb-8710-22eaeba6c00e.png)